### ISSUE TITLE
Can't see Check in Checkbox with Firefox in High Contrast Mode

### ISSUE THREAD

**Issue by lupine86** [OfficeDev/office-ui-fabric-react#1946]
### Bug Report
- __Package version(s)__: office-ui-fabric-react@2.34.2
- __Browser and OS versions__: Firefox

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? Yes

Requested priority: Normal

Products/sites affected: VSO VSTS

#### Describe the issue:

The checkbox does not show the check in high contrast mode when using fire fox

#### Actual behavior:

The checkbox is filled with a solid color but you can't see the check

#### Expected behavior:

You should be able to see the check

### If applicable, please provide a codepen repro:

https://codepen.io/lupine86/pen/MowjBp

This is likely caused by the ::before style setting a 10px border.  Should this be 1px?
https://github.com/OfficeDev/office-ui-fabric-react/blob/da0f88a8690c867e2bbcf19c533ba57487acc4ed/packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.scss#L139

![image](https://user-images.githubusercontent.com/11162250/26839894-5a304374-4ab2-11e7-932a-3e3d4863cfc7.png)
